plugins/stack-overflow-backend/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @backstage/plugin-stack-overflow-backend
 
+## 0.1.2
+
+### Patch Changes
+
+- Added apiKey configuration to prevent encoding the key value
+
```

Comment:
Note: we don't update the changelog manually, there is a guide [here](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets) explaining that if you create a changeset file, the changelog and also the version in the package.json file will be updated automatically 🙂.
Could you remove these changes and create a changeset?